### PR TITLE
feat: Signing modal consistency

### LINKED ISSUE #8904: Signing modal consistency
Since there are currently 3 modals for signing - claim free drop, autoteleport signing, simple action signing  - Let's make it more consistent and use fewer components

Less is more, we are getting rid of the infinite loading and also the tips and facts about kodadot (they are not updated anyway) 
- Our modal is also rescaling with every next tip and fact, it is not pleasant tbh

# Design 
image
- created new page in handoff: https://www.figma.com/file/JXNtryaNYhY0XJNArzvPQx/Handoff?type=design&node-id=6886-20058&mode=design&t=Z8Nase54i6ZXvZca-11
- I added some notes, so it's not needed to find this issue every time 
- Reuses our current free drop and paid drop claim modal
- Will be used for every action that requires only one signing - buying, minting, offer, transfer, listing, etc.

## Good to know 
- there is not green check, when the transaction is confirmed, it simply goes to a success message 
   _- I will create them gradually over the next week_
- right now it will just disappear and display our old congrats for actions that don't have new one yet 
- grey box is not skeleton loader: its a solid color (neutral-3, in dark mode neutral-11)
- Transaction type is displayed as a heading - currently “Minting nft” - it is replaced based on what you are signing, e.g. Transfer 10 KSM

# Signing modal - Multiple
- we already have this one, it's used anywhere where autoteleport is enabled 
- the addition over the simple modal is the message, and two additional parts - check funds and teleport 
image
- Green checks are displayed when a step is completed, after all steps are completed, the Main button is active with the label “continue” 
- After you click continue, it redirects to a success modal

### PR DESCRIPTION
**Pull request by hassnian**
## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #8904

new signing modal added to:

 - create nft 
 - create collection
 - buy modal
 - teleport
 - transfer nft
 - listing modal
 - holder of generative
 - burn nft
 - burn collection
 - burn all nfts of collection

## Needs Design check

- @exezbcz please review

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; 

CleanShot 2024-01-15 at 15 13 38@2x

### CODE REVIEW

**Comment by exezbcz** [comment]
question: is it also done so that when you are mass listing, it adds "s" to the copy 


image


- so there would be listing NFTs

**Comment by hassnian** [comment]
> question: is it also done so that when you are mass listing, it adds "s" to the copy
> 
> image
> * so there would be listing NFTs

yes

File: components/shared/SigningModal/SigningModalBody.vue [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <div class="py-5 flex items-start">
+      <NeoIcon icon="lightbulb" size="small" class="mr-2 is-block" />
+      <p
+        v-dompurify-html="$t('autoTeleport.tip')"
+        class="is-size-7 capitalize" />
+    </div>
+  </slot>
+
+  <div class="min-height">
+    <SkeletonLoader
```
Comment (preschian) [inline components/shared/SigningModal/SigningModalBody.vue:16]:
```suggestion
  <div class="min-h-[200px]">
```

**Review by preschian — APPROVED** [review]
nice 👍, code lgtm

(not tested)

File: components/shared/SigningModal/SigningModalBody.vue (hunk 1 above)
Comment (hassnian) [reply components/shared/SigningModal/SigningModalBody.vue:16]: fixed 
CleanShot 2024-01-16 at 18 28 52@2x

**Comment by exezbcz** [comment]
will test soon

**Comment by yangwao** [comment]
I would release it next week to be rigorously tested as prury is away

**Review by roiLeo — COMMENTED** [review]
linter not happy, could you check

**Comment by yangwao** [comment]
cc @exezbcz to drop visual-ok and we can merge

**Comment by exezbcz** [comment]
failed to sign not working? 

image


when i cancelled signing it just closed the modal

**Comment by exezbcz** [comment]
bullish, 

teleport works: 

image


- weird, it did not went through for the first time? looks like issue is elsewhere 

https://github.com/kodadot/nft-gallery/issues/9073



minting collection: 

image

- cancelling transaction still does not work 
image
image

**Comment by hassnian** [comment]
https://github.com/kodadot/nft-gallery/assets/44554284/10524b55-9357-4203-982f-606c05a4f6ab

@roiLeo @preschian added try again feature , review again if needed 

> * cancelling transaction still does not work

@exezbcz  was not added, pls try now

**Comment by yangwao** [comment]
we can hold this one for this week release as it's not well tested and we don't want to break something.

**Comment by yangwao** [comment]
I think it's over drop and we can push it to beta so we can collectively test out

Otherwise it would needs consistently maintain conflicts

Agree?

**Comment by yangwao** [comment]
Thanks!
pay 90 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $90 @ 6.56 USD/DOT ~ 13.72 $DOT
🧗 [13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh](https://polkadot.subscan.io/account/13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh)
🔗 [0x4a8b241b2d340e2597ce769dbf5ef06a6a14b6114f3f2079c3c1cdf7a4c8b812](https://polkadot.subscan.io/extrinsic/0x4a8b241b2d340e2597ce769dbf5ef06a6a14b6114f3f2079c3c1cdf7a4c8b812)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**